apis/python/tests/test_basic_anndata_io.py

```diff
@@ -988,7 +991,7 @@ def make_uns_adata(
     tmp_path: Path,
     measurement_name: str = "RNA",
     uns: UnsMapping | None = None,
-) -> Tuple[str, AnnData]:
+) -> tuple[str, AnnData]:
```

Comment:
In future these kinds of things could be a separate PR -- very nice catch (thank you!) but not essential to the primary purpose of this PR

Reply:
Done: #3769